src/components/EntryForm.js

```diff
@@ -0,0 +1,359 @@
+                className={styles.questionnaireCol}
+                type="radio"
+                value={false}
+                aria-labelledby="no"
+                defaultChecked={loo[q.property] === false}
+                data-testid={`${q.property}:no`}
+              />
+
+              <input
+                ref={register}
+                name={q.property}
+                className={styles.questionnaireCol}
+                type="radio"
+                value=""
+                aria-labelledby="unknown"
+                defaultChecked={loo[q.property] === ''}
+                data-testid={`${q.property}:unknown`}
+              />
+            </fieldset>
+          ))}
+        </div>
+
+        <label>
+          Fee?
+          <input
+            ref={register}
+            name="fee"
+            type="text"
+            className={controls.text}
+            defaultValue={loo.fee === null ? '' : loo.fee}
+            placeholder="The amount e.g. £0.10"
+            data-testid="fee"
+          />
+        </label>
+
+        <label>
+          Any notes?
+          <textarea
+            ref={register}
+            name="notes"
+            className={controls.text}
+            defaultValue={loo.notes === null ? '' : loo.notes}
+            data-testid="notes"
+          />
+        </label>
+
+        <h3 className={headings.regular}>Data for this toilet</h3>
+
+        <div className={styles.data}>
+          <label>
+            Latitude
+            <input
+              ref={register}
+              type="text"
+              name="geometry.coordinates.0"
+              className={controls.text}
+              defaultValue={center.lat}
+              readOnly={true}
+            />
+          </label>
+
+          <label>
+            Longitude
+            <input
+              ref={register}
+              type="text"
+              name="geometry.coordinates.1"
+              className={controls.text}
+              data-testid="loo-name"
+              defaultValue={center.lng}
+              readOnly={true}
+            />
+          </label>
+        </div>
+
+        {props.saveLoading && <Loading message="Saving your changes..." />}
+
+        {
+          // This message probably won't be seen, but just in case the redirect fails...
+          saveResponse && saveResponse.submitReport.code === '200' && (
+            <Notification>
+              Successfully saved changes! Redirecting&hellip;
+            </Notification>
+          )
+        }
+
+        {
+          // TODO better error message?
+          (saveError ||
+            (saveResponse && saveResponse.submitReport.code !== '200')) && (
+            <Notification>
+              Oops, there was an error saving your changes.
+              {console.log(saveError, saveResponse)}
+            </Notification>
+          )
+        }
+
+        <div className={controls.btnStack}>
+          {isFunction(children) ? children({ dirtyFields }) : children}
```

Comment:
Allow children to be render prop to set `disabled` on the `EditPage` when no fields changed.
Intended to replce [onlyChanges](https://github.com/neontribe/gbptm/pull/588/files#diff-332fe77b4fa1c3806c14e30751be664fL547) :crossed_fingers: